src/Misc/layoutbin/installdependencies.sh

```diff
@@ -36,7 +36,17 @@ then
         command -v apt
         if [ $? -eq 0 ]
         then
-            apt update && apt install -y liblttng-ust0 libcurl3 libkrb5-3 zlib1g
+            apt update && apt install -y liblttng-ust0 libkrb5-3 zlib1g
+            if [ $? -ne 0 ]
+            then
+                echo "'apt' failed with exit code '$?'"
+                print_errormessage
+                exit 1
+            fi
+            
+            # ubuntu 18 uses libcurl4
+            # ubuntu 14, 16 and other linux use libcurl3
+            apt install -y libcurl4 || apt install -y libcurl3
```

Comment:
do the reverse older, `libcurl3 || libcurl4` like how we do `libicu`
you also need to change the `apt-get` part on line 79 of this file.